### PR TITLE
Added Container-Interop adapter

### PR DESCRIPTION
**Pull request by shochdoerfer**
An adapter for the Container-Interop interface was created to be able to measure how long the object instantiation takes.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -17,7 +17,8 @@
     },
     "require": {
         "psr/log": "~1.0",
-        "php" : "~5.4"
+        "php" : "~5.4",
+        "container-interop/container-interop": "^1.1"
```
Comment (fabfuel) [inline composer.json]: This is a pretty specialized adapter, please move the requirement to "require-dev". 
Projects which use container-interpop will have it as its own dependency already.

File: src/Fabfuel/Prophiler/Adapter/Interop/Container/Container.php [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+ * @author @shochdoerfer <S.Hochdoerfer@bitExpert.de>
+ * @created 18.10.15, 12:14
+ */
+namespace Fabfuel\Prophiler\Adapter\Interop\Container;
+
+use Fabfuel\Prophiler\Adapter\AdapterAbstract;
+use Fabfuel\Prophiler\ProfilerInterface;
+use Interop\Container\ContainerInterface;
+
+class Container extends AdapterAbstract implements ContainerInterface
```
Comment (fabfuel) [inline src/Fabfuel/Prophiler/Adapter/Interop/Container/Container.php]: Please add a unit test for your adapter and please have a look at the Scrutinizer issues, mentioned here:
https://scrutinizer-ci.com/g/fabfuel/prophiler/inspections/c8e11dda-bae4-4a30-a391-9da6d35e69e4/issues/

**Comment by shochdoerfer** [comment]
Anything else needed from my side?

**Comment by fabfuel** [comment]
No, all good. Thanks a lot!

**Comment by shochdoerfer** [comment]
Would you mind tagging a new release? Would love to throw away my own copy of the code ;)

**Comment by shochdoerfer** [comment]
Also, I just made a PR for the container-interop repo to list Prophiler as a project using the container-interop interface, see [here](https://github.com/container-interop/container-interop/pull/48).

**Comment by fabfuel** [comment]
Sure, I just tagged version 1.5.0

**Pull request merged**